frappe/public/js/frappe/utils/datetime.js

```diff
@@ -98,23 +98,28 @@ $.extend(frappe.datetime, {
 		return moment().endOf("year").format();
 	},
 
-	get_user_fmt: function() {
```

Comment:
The new methods `get_user_date_fmt` and `get_user_time_fmt` are better but we can't lose this old method as it will be a breaking change.

Reply:
Previous method `get_user_fmt` will be restored (with a comment indicating it is for backwards compatibility only).